src/Routes/Signatures/Routes.js

```diff
@@ -1,16 +0,0 @@
-import React, { Suspense, lazy } from 'react';
-import { Route, Switch } from 'react-router-dom';
-
-import Loading from '../../Components/Loading/Loading';
-
-const List = lazy(() => import(/* webpackChunkName: 'Signature List' */ './Signatures'));
-const Details = lazy(() => import(/* webpackChunkName: "Signature Details" */ './Details'));
-
-const SigRoutes = () => <React.Fragment>
-    <Switch>
-        <Route exact path='/signatures' component={() => <Suspense fallback={<Loading />}> <List /> </Suspense>} />
-        <Route exact path='/signatures/:id' component={() => <Suspense fallback={<Loading />}> <Details /> </Suspense>} />
-    </Switch>
-</React.Fragment>;
-
-export default SigRoutes;
```

Comment:
With v6 it should still be possible to use these sub route components. The only change that would be required is to change from `Switch` to `Routes` and make the paths relative (removing the `/` at the start).

See https://reactrouter.com/en/main/upgrading/v5#upgrade-all-switch-elements-to-routes

Reply:
I do not see any benefits of having sub-routes. Thus, I prefer to remove them. In case, if I miss any let me know. Happy reconsider.